### PR TITLE
framework/st_things : modify to save cloud data unnecessarily many times.

### PR DESCRIPTION
**Pull request by JeongJunSik**
- st_things Logic modification to save state to file (fread, data size argument in fwrite)
- code refactoring : save cloud data uncessarily many times.
                                remove default ci server address.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [e1c40e43f1880e30769ea272b5254fc4bc6e8386] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [e1c40e43f1880e30769ea272b5254fc4bc6e8386] - Code Rule Check (C++) OK.

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c [hunk 1]
```diff
@@ -918,7 +918,7 @@ int esm_save_easysetup_state(int state)
 			return 0;
 		}
 		unsigned char easysetup_completed = ES_COMPLETE;
-		int ret = fwrite(&easysetup_completed, ES_COMPLETE, sizeof(unsigned char), fp);
+		int ret = fwrite(&easysetup_completed, 1, sizeof(unsigned char), fp);
```
Comment (juitem) [inline framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: This is bug fix, I'd rather suggest change the title. Edit->Fix

**Comment by seinfra** [comment]
Target : [b34922fda7da535f3b055145dfe5c7fcc66b574e] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b34922fda7da535f3b055145dfe5c7fcc66b574e] - Code Rule Check OK.

**Pull request merged**